wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md

```diff
@@ -0,0 +1,45 @@
+---
+tags:
+  - medals
+  - medal hunting
+---
+
+# osu! Medal Hunters Discord server
+
+::: Infobox
+
+#### osu! Medal Hunters
+
+![Server logo](img/logo.png "The Medal Hunter Discord server logo, designed by Tanza3D")
+
+|  |  |
+| :-- | :-- |
+| Invite link | <https://discord.gg/osumedalhunters> |
+| Website | <https://osekai.net/> |
+
+:::
+
+*Notice: While working together in order to find Hush-Hush medals is encouraged, finding them by yourself is as well. Be advised that the website and server contain solutions to Hush-Hush medals.*
+
+**osu! Medal Hunters** is a the main [Discord](https://discord.com) server used by the medal hunting community, where people discuss [medals](/wiki/Medals) and [medal solutions](/wiki/Medals/Unlock_requirements).
+
+The server is run by the same team that develops and maintains the [Osekai website](https://osekai.net/), which is a hub for alternative rankings, medal solutions, archived osu! client versions, and a [profile badge](/wiki/Community/Profile_badge) database.
```

Comment:
mixed feelings due to https://github.com/ppy/osu-wiki/pull/5736#issuecomment-866580536

Reply:
more mixed feelings due to https://github.com/ppy/osu-wiki/pull/7949#discussion_r962355297

Reply:
i'm gonna require @Ephemeralis's opinion here. i put a warning at the top of the page and i don't know exactly how more gracefully i should word this.

Reply:
I guess we can let it pass since a) it doesn't mention the medals which shall not be named, and b) it's an open secret anyway, but I'd still run it past @peppy just to get an explicit approval/rejection

Reply:
a) well sorta

again, as i said i put a very explicit notice at the top of the page

> *Notice: While working together in order to find Hush-Hush medals is encouraged, finding them by yourself is as well. Be advised that the website and server contain solutions to Hush-Hush medals.*

Reply:
No.

In fact I'm not sure this article should exist at all? Are we suddenly allowing third party discords to advertise on the wiki?

Reply:
assuming that only concerns this article (as it has a reason), since some other servers also have closely related websites (e.g. osu! storyboarder banquet with storyboarder.xyz, and osu! University with its youtube, spotify and twitter etc.)

Reply:
I dunno. Highlight me before linking external sites and we'll handle on a per-case basis.

Reply:
I went ahead and removed the website mentions [`9ab273d` (#8152)](https://github.com/ppy/osu-wiki/pull/8152/commits/9ab273d9bb7901a39c05ef10b2f3a43c40705c14), hope that works better